client/src/pages/SubmissionDetails.vue

```diff
@@ -95,7 +132,12 @@
           </q-list>
         </div>
         <div v-else>
-          <q-list bordered separator data-cy="list_assigned_reviewers">
+          <q-list
```

Comment:
Change the list item to a q-card

Reply:
This is a list of reviewers, so it needs to remain as a list.